src/services/ApiService/MembershipApi.ts

```diff
@@ -16,15 +16,15 @@ export const formatResponse = (timestamps: PromiseType<ReturnType<typeof getUnlo
   // Convert to Bigies so we can determine their expiry date.
   // @todo: prefer date-fns for time comparaisons.
   const expiries = pipe(
-    mapObjIndexed(mapObjIndexed(toString)),
-    mapObjIndexed(mapObjIndexed(bigify))
+    mapObjIndexed(mapObjIndexed(bigify)),
+    mapObjIndexed(mapObjIndexed(toString))
   )(timestamps);
 
   return Object.keys(expiries)
     .map((address: TAddress) => ({
       address,
       memberships: Object.keys(expiries[address])
-        .filter((contract) => expiries[address][contract].isGreaterThan(bigify(0)))
+        .filter((contract) => expiries[address][contract] !== '0')
```

Comment:
This is not equivalent

Reply:
It doesn't have to be since we're just eliminating null values.
The contract api set's null as `0x00`.
`expiries` transforms it to a string '0' on which we can filter.